enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -149,21 +198,28 @@ private Cluster startCluster( String recordFormat )
                 .withSharedCoreParam( GraphDatabaseSettings.record_format, recordFormat )
                 .withSharedReadReplicaParam( GraphDatabaseSettings.record_format, recordFormat );
         Cluster cluster = clusterRule.startCluster();
-        createSomeData( clusterDatabase( cluster ) );
+        createSomeData(  cluster );
         return cluster;
     }
 
-    public static DbRepresentation createSomeData( GraphDatabaseService db )
+    public static DbRepresentation createSomeData( Cluster cluster )
     {
-        try ( Transaction tx = db.beginTx() )
+        try
         {
-            Node node = db.createNode();
-            node.setProperty( "name", "Neo" );
-            node.setProperty( "random", Math.random() * 10000 );
-            db.createNode().createRelationshipTo( node, RelationshipType.withName( "KNOWS" ) );
-            tx.success();
+            cluster.coreTx( ( coreGraphDatabase, transaction ) ->
+            {
+                Node node = coreGraphDatabase.createNode( label );
+                node.setProperty( PROP_NAME, "Neo" );
+                node.setProperty( PROP_RANDOM, Math.random() * 10000 );
+                coreGraphDatabase.createNode().createRelationshipTo( node, RelationshipType.withName( "KNOWS" ) );
+                transaction.success();
+            } );
+        }
+        catch ( Exception e )
+        {
+            e.printStackTrace();
```

Comment:
Just let the exception through. The coreTx(..) method will retry on any transient failures. Any other exceptions should be considered issues and bubble up to fail the test.

Reply:
Actually this code is called within a Thread, so it is more convenient to wrap it here than to have the method declare it throws and have to wrap it anyway in the Thread declaration. Not addressed, but used `launderedException` instead of `printStackTrace`